[{"name": "test.test_perakkaisten_summa.SummaTest.test_0", "status": "passed", "message": "", "passed": true, "points": ["3.perakkaisten_summa_versio2"], "backtrace": []}, {"name": "test.test_perakkaisten_summa.SummaTest.test_1", "status": "failed", "message": "False is not true : Ohjelmasi pitäisi tulostaa\nLaskettiin 1 + 2 = 3\nsyötteellä 3. Ohjelmasi tulosti\n3", "passed": false, "points": ["3.perakkaisten_summa_versio2"], "backtrace": ["  File \"/usr/lib/python3.8/unittest/case.py\", line 60, in testPartExecutor\n    yield\n", "  File \"/usr/lib/python3.8/unittest/case.py\", line 676, in run\n    self._callTestMethod(testMethod)\n", "  File \"/usr/lib/python3.8/unittest/case.py\", line 633, in _callTestMethod\n    method()\n", "  File \"/home/notmythingy/Programming/mooc-ohjelmointi-21/osa03-05_perakkaisten_summa/test/test_perakkaisten_summa.py\", line 45, in test_1\n    self.assertTrue(sanitize(odotettu) == sanitize(output), f\"Ohjelmasi pitäisi tulostaa\\n{odotettu}\\nsyötteellä {luku}. Ohjelmasi tulosti\\n{output}\")\n", "  File \"/usr/lib/python3.8/unittest/case.py\", line 765, in assertTrue\n    raise self.failureException(msg)\n"]}, {"name": "test.test_perakkaisten_summa.SummaTest.test_2", "status": "passed", "message": "", "passed": true, "points": ["3.perakkaisten_summa_versio2"], "backtrace": []}]